pages/assets/[id].tsx

```diff
@@ -48,14 +49,15 @@ const Asset: React.FC<Props> = () => {
   }, []);
 
  if (asset && asset.status === 'errored') {
-    const message = asset.errors && asset.errors.messages[0];
-    errorMessage = `Error creating this asset: ${message}`;
+    errorMessage = 'Error creating this asset: Please upload a valid video file (gifs are not supported)'
+    errorDetails = asset.errors && asset.errors.messages[0];
   }
 
   if (errorMessage) {
     return (
       <Layout darkMode={false}>
         <div><h1>{errorMessage}</h1></div>
+        {errorMessage && <p>{errorDetails}</p>}
```

Comment:
@alexdechaves whoops, I just caught this, it should be:

```
{errorDetails && <p>{errorDetails}</p>}
```

Reply:
i literally changed it back from what you gave me :(